doc/modules/tree.rst

```diff
@@ -375,6 +375,10 @@ Tips on practical use
     predict. Training time can be orders of magnitude faster for a sparse
     matrix input compared to a dense matrix when features have zero values in
     most of the samples.
+    
+  * In case that there are multiple tied classes with a joint highest 
+    probability, the classifier will predict the class with the lowest index
+    amongst those classes.
```

Comment:
```suggestion
  * In case that there are multiple classes with the same highest joint 
    probability, the classifier will predict the class with the lowest index
    amongst those classes.
```

Reply:
This definitely makes it clearer